django/applications/catmaid/control/cropping.py

```diff
@@ -524,27 +510,20 @@ def rotate2d(degrees, point, origin) -> Tuple[float, float]:
     return newx, newyorz
 
 @task()
-def process_crop_job(job, create_message=True) -> str:
+def process_crop_job(job: CropJob, create_message=True) -> str:
     """ This method does the actual cropping. It controls the data extraction
     and the creation of the sub-stack. It can be executed as Celery task.
     """
     try:
         # Create the sub-stack
         cropped_stack = extract_substack(job)
-
-        # Create tho output image
-        outputImage = ImageList()
-        for img in cropped_stack:
-            outputImage.append( img )
-
         # Save the resulting micro_stack to a temporary location
         no_error_occured = True
         error_message = ""
         # Only produce an image if parts of stacks are within the output
-        if len( cropped_stack ) > 0:
-            outputImage.writeImages( job.output_path.encode('ascii', 'ignore') )
-            # Add some meta data to the image
-            addMetaData( job.output_path, job, cropped_stack )
+        if len(cropped_stack) > 0:
+            metadata = job.create_tiff_metadata(len(cropped_stack))
+            cropped_stack[0].save(job.output_path.encode("ascii", "ignore"), compression="raw", save_all=True, append_images=cropped_stack[1:], tiffinfo=metadata)
```

Comment:
This line does need some changes too. First, the format needs to be specified and the ASCII conversion doesn't seem to be needed for me:
```
cropped_stack[0].save(job.output_path, 'TIFF', compression="raw",
    save_all=True, append_images=cropped_stack[1:], tiffinfo=metadata)
```

Reply:
It turns out you don't need the format ('TIFF'), but it doesn't understand the extension with the ASCII string, but this works:
```
cropped_stack[0].save(job.output_path, compression="raw",
    save_all=True, append_images=cropped_stack[1:], tiffinfo=metadata)
```

Reply:
Since I merged this branch already, I fixed this in `dev`.